compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/SkikoUIView.kt

```diff
@@ -462,7 +462,8 @@ internal class SkikoUIView : UIView, UIKeyInputProtocol, UITextInputProtocol {
         val p = (position as? IntermediateTextPosition)?.position ?: return null
         val endOfDocument = input?.endOfDocument()
         return if (endOfDocument != null) {
-            IntermediateTextPosition(max(min(p + offset, endOfDocument), 0))
+            val result = input?.positionFromPosition(position = p, offset = offset)
+            IntermediateTextPosition(result ?: max(min(p + offset, endOfDocument), 0))
```

Comment:
Should we really do such fallback here?
If so, use `coerceIn` instead of `min`/`max` pair

Reply:
Good suggestion

Reply:
Done. Changed to coerceIn